data/json/recipes/recipe_weapon.json

```diff
@@ -215,6 +229,19 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "leather", 20 ] ], [ [ "rope_superior_short", 2, "LIST" ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bullwhip_razor",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "4 h",
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ]
```

Comment:
The number of opened brackets is not equal the number of closed.

Reply:
thanks